lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/filteredStringListView.py

```diff
@@ -65,101 +67,79 @@ def paint(
         painter.fillRect(event.rect(), Theme.instance().palette.colorListBorder)
         super(FilteredStringListView, self).paintEvent(event)
-        if self._model.isFilteredEmpty():
-            painter = QPainter(self.viewport())
-            painter.setPen(Theme.instance().palette.colorPlaceHolderText)
-            painter.drawText(self.rect(), Qt.AlignCenter, kNoObjectFoundLabel)
-
-    @property
-    def items(self) -> Sequence[str]:
-        return self._model.stringList
-
-    @items.setter
-    def items(self, items: Sequence[str]):
-        # save the selected items
-        prevSelectedIndices = []
-        for s in self.selectedItems:
-            try:
-                newIdx = items.index(s)
-            except ValueError:
-                continue
-            prevSelectedIndices.append(newIdx)
-
-        self._model.setStringList(items)
-        # restore the selected items
-        newSelection = QItemSelection()
-        for i in prevSelectedIndices:
-            idx = self._model.index(i, 0)
-            newSelection.select(idx, idx)
-        self.selectionModel().select(newSelection, QItemSelectionModel.ClearAndSelect)
-        self.updatePlaceholder()
-
-    @property
+
+        model = self.model()
+        if not model:
+            return
+
+        if model.isFilteredEmpty():
+            self._paintPlaceHolder(kNoObjectFoundLabel)
+        elif model.rowCount() == 0:
+            if Host.instance().canDrop and Host.instance().canPick:
+                self._paintPlaceHolder(kDragOrPickObjectsLabel)
+            elif Host.instance().canDrop:
+                self._paintPlaceHolder(kDragObjectsHereLabel)
+            elif Host.instance().canPick:
+                self._paintPlaceHolder(kPickObjectsLabel)
+
     def selectedItems(self) -> Sequence[str]:
         return [str(index.data(Qt.DisplayRole)) for index in self.selectedIndexes()]
 
-    @property
     def hasSelectedItems(self) -> bool:
-        return self.selectionModel().hasSelection()
-
-    def updatePlaceholder(self):
-        """Show or hide placeholder based on the model's content."""
-        if self.placeholder_label is not None:
-            model = self.model()
-            if model and model.rowCount() == 0:
-                self.placeholder_label.show()
-            else:
-                self.placeholder_label.hide()
-
-    def resizeEvent(self, event):
-        """Ensure placeholder is centered on resize."""
-        super().resizeEvent(event)
-        if self.placeholder_label is not None:
-            self.placeholder_label.setGeometry(self.viewport().geometry())
+        return bool(self.selectionModel().hasSelection())
+
+    def _paintPlaceHolder(self, placeHolderText):
+        painter = QPainter(self.viewport())
+        theme = Theme.instance()
+        painter.setPen(theme.palette.colorPlaceHolderText)
+        font: QFont = painter.font()
+        font.setPixelSize(theme.uiScaled(18))
+        painter.setFont(font)
```

Comment:
this was removed as of Scott's advice

Reply:
You mean he did not want a larger font? I can remove the font size

Reply:
yes, no larger font - the same font size is requested as everywhere else.